pom.xml

```diff
@@ -224,6 +240,16 @@
              <role>developer</role>
            </roles>
         </developer>
+		<developer>
+			<name>Luigi Andrea Pascarelli</name>
+			<email>luigiandrea.pascarelli@4science.it</email>
+			<timezone>GMT +1</timezone>
+		</developer>
+		<developer>
+			<name>Francesco Pio Scognamiglio</name>
+			<email>francescopio.scognamiglio@4science.it</email>
+			<timezone>GMT +1</timezone>
+		</developer>
```

Comment:
As this is a generated API, we shouldn't be adding <developer> tags here for individuals. So these two tags should be removed in my opinion. If you notice, we only have two developer tags here right now to describe that the XSDs are owned by ORCID, and the Committers maintain the JAXB configs only.